cc/main.cc

```diff
@@ -456,13 +465,21 @@ class SelfPlayer {
       gflags::SetCommandLineOption(flag_value.first.c_str(),
                                    flag_value.second.c_str());
     }
+
+    if (FLAGS_resign_threshold < old_resign && old_resign == 1.0) {
+      // NOTE: This will trigger an interupt (and reload) of all current games.
+      required_flags_ = flags_timestamp_
+    }
   }
 
   absl::Mutex mutex_;
   std::unique_ptr<DualNetFactory> dual_net_factory_ GUARDED_BY(&mutex_);
   Random rnd_ GUARDED_BY(&mutex_);
   std::vector<std::thread> threads_;
+  // Timestamp of newest flag file.
   uint64_t flags_timestamp_ = 0;
+  // Signals all currently playing games to stop and restart with new flags.
+  uint64_t required_flags_ = 0;
```

Comment:
`required_flags_` sounds like it could be a set of flags or something. How about something like `required_flags_timestamp_`?

Reply:
I shortened it to avoid a line wrap. renamed per suggestion.